gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiPlansResource.java

```diff
@@ -180,7 +182,7 @@ public Response createApiPlan(@Valid @NotNull CreateGenericPlan createPlan) {
                 new CreatePlanUseCase.Input(
                     apiId,
                     api -> planMapper.map(planV4, api),
-                    api -> flowMapper.map(planV4.getFlows(), api),
+                    api -> ofNullable(flowMapper.map(planV4.getFlows(), api)).orElse(emptyList()),
```

Comment:
What do you think about adding a test to validate the fix?

Reply:
@jourdiw I have a strong opinion that we shouldn't expect null values in domain code. And if we _need_ to pass null value, we should use Optionals instead. WDYM?

Reply:
Yes that's fine for me 👍